lib/routemap.c

```diff
@@ -2303,9 +2303,9 @@ DEFUN (no_match_metric,
       if (argc <= idx_number)
         return rmap_match_set_hook.no_match_metric (vty, index, "metric",
                                                     NULL, RMAP_EVENT_MATCH_DELETED);
-        return rmap_match_set_hook.no_match_metric(vty, index, "metric",
-                                                   argv[idx_number]->arg,
-                                                   RMAP_EVENT_MATCH_DELETED);
+      return rmap_match_set_hook.no_match_metric(vty, index, "metric",
```

Comment:
That's the whole point @qlyoung some compilers are now warning when the indentation level does not match.  This fixes that issue and puts the return statement at the right spot